src/main/kotlin/no/fdk/concept_catalog/controller/ImportController.kt

```diff
@@ -46,22 +44,25 @@ class ImportController(@Qualifier("import-executor") private val importExecutor:
         }
     }
 
-    @PutMapping(value = ["/{importId}/confirm"])
-    fun confirmImport(
+    @PutMapping(value = ["/{importId}/confirmConceptImport"])
```

Comment:
Vi har i stor grad holdt oss til kebab case i path-segmenter, tror det er best practice

```suggestion
    @PutMapping(value = ["/{importId}/confirm-concept-import"])
```